_posts/2023-04-15-announcing-ballerina-2201.5.0-swan-lake-update-5.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog-post
+title: Announcing Ballerina 2201.5.0 (Swan Lake Update 5)
+author: Ballerina Team
+published-date: 15 April 2023
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2023-04-15-announcing-ballerina-2201.5.0-swan-lake-update-5/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are excited to announce that Ballerina Swan Lake has just released its fifth update release, [Ballerina 2201.5.0 (Swan Lake Update 5)](https://ballerina.io/downloads/). In addition to the existing macOS installer, this release includes a [new installer](https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg) specifically designed for installing Ballerina on your macOS-ARM platforms.
```

Comment:
Is the first sentence correct? On the doc, I suggested 

"We are excited to announce the fifth update release of Ballerina Swan Lake," 

Or we can say

"The Ballerina team is excited to announce the fifth update release of Ballerina Swan Lake,"

Reply:
What is incorrect in the current sentence? @MaryamZi

Reply:
It reads like "Ballerina Swan Lake has released a new version of itself" to me :)

Reply:
It is clearly mentioned that "Ballerina Swan Lake has released its fifth update release".

Reply:
@anupama-pathirage @sameerajayasoma - WDYT?

Reply:
> It is clearly mentioned that "Ballerina Swan Lake has released its fifth update release".

My point is that this is like saying "Ballerina Swan Lake has released a new version of itself".

Reply:
Updated.